src/widgets/chart/lv_chart.c

```diff
@@ -1073,6 +1084,9 @@ static void draw_series_bar(lv_obj_t * obj, lv_layer_t * layer)
     int32_t y_tmp;
     lv_chart_series_t * ser;
     uint32_t ser_cnt = lv_ll_get_len(&chart->series_ll);
+    if(ser_cnt == 0) {
+        return;
+    }
     int32_t block_gap = lv_obj_get_style_pad_column(obj, LV_PART_MAIN);  /*Gap between the column on ~adjacent X*/
     int32_t block_w = (w - ((chart->point_cnt - 1) * block_gap)) / chart->point_cnt;
```

Comment:
I tried to see if this was also a zero-division risk but it's not, since the setter for `chart->point_cnt` ensures it's at least 1 :slightly_smiling_face: